tests/functional/n/no/no_member_property_decorator.py

```diff
@@ -0,0 +1,38 @@
+"""Tests for no-member on decorated properties.
+
+https://github.com/PyCQA/pylint/issues/1127
+"""
+# pylint: disable=missing-function-docstring, too-few-public-methods, invalid-name
+# pylint: disable=deprecated-module, missing-class-docstring
+
+from unittest import TestCase
+
+class prop:
+    def __init__(self):
+        self.value = 5
+
+class my_class:
+    @prop
+    def my_property(self):
+        pass
+
+print(my_class().my_property.value)
+
+
+def with_value(Cls):
+    Cls_setUp = Cls.setUp
```

Comment:
This feels really unpythonic. Can we make the tests a little more pythonic name- and code-wise?

Reply:
Sure, tho it's worth noting I copied it directly from the issue to demonstrate that it works.